docs/sources/whatsnew/whats-new-in-v10-1.md

```diff
@@ -39,3 +39,20 @@ You can now choose whether to set a threshold above which values in the data sho
 To learn more, refer to our [disconnect values documentation]({{< relref "../panels-visualizations/visualizations/time-series/#disconnect-values" >}}).
 
 {{< figure src="/media/docs/grafana/screenshot-grafana-10-1-disconnect-values-examples.png" max-width="750px" caption="Disconnect values in time series, trend, and state timeline visualizations" >}}
+## Authentication: Configure refresh token handling separately for OAuth providers
+
+<!-- Mihaly Gyongyosi -->
+
+_Available in public preview in all editions of Grafana._
+
+With Grafana v9.3 we introduced a feature toggle called `accessTokenExpirationCheck` to improve the security of Grafana by checking the expiration of the access token and automatically refreshing the expired access token when the user is logged in using one of the OAuth providers.
+
+With the current release, we introduce a new configuration option for each OAuth provider called `use_refresh_token` that allows you to configure whether the particular OAuth integration should use refresh tokens to automatically refresh the access tokens when it expires. Besides, to further improve the security and provide secure defaults we enabled `use_refresh_token` by default for providers that either support refresh tokens by default or fetching refresh tokens is controlled by the client. It is enabled by default for the following OAuth providers: `AzureAD`, `GitLab`, `Google`.
```

Comment:
This part in the second sentence isn't clear: "refresh tokens by default" - here, do you mean the type of token or the action of refreshing?


```suggestion
With the current release, we've introduced a new configuration option for each OAuth provider called `use_refresh_token`, which allows you to configure whether the particular OAuth integration should use refresh tokens to automatically refresh the access tokens when it expires. In addition, to further improve the security and provide secure defaults, we've enabled `use_refresh_token` by default for providers that support either refresh tokens by default or fetching refresh tokens controlled by the client. It is enabled by default for the following OAuth providers: `AzureAD`, `GitLab`, `Google`.
```

Reply:
Hello @imatwawana. I believe @mgyongyosi means the action of refreshing the tokens.

Would it be better rephrased as
```
In addition, to further improve the security and provide secure defaults, we've enabled `use_refresh_token` by default for providers that support either refreshing tokens automatically or fetching refresh tokens controlled by the client. It is enabled by default for the following OAuth providers: `AzureAD`, `GitLab`, `Google`.
```

Let me know what you think.

Reply:
Yes, that works. Also I still want to rephrase that whole sentence a bit. The second part of the sentence wasn't quite right either. I'm thinking:

`
In addition, to further improve the security and provide secure defaults, `use_refresh_token` is enabled by default for providers that support either refreshing tokens automatically or client-controlled fetching of refresh tokens. 
`

Reply:
Added!